Assets/MixedRealityToolkit/Utilities/AssetScriptReferenceRetargeter.cs

```diff
@@ -343,10 +364,164 @@ private static bool IsYamlFile(string filePath)
             }
         }
 
-        private static void CleanupDirectory(string outputPath)
+        private static void PostProcess(string outputPath)
         {
             DirectoryInfo outputDirectory = new DirectoryInfo(outputPath);
             RecursiveFolderCleanup(outputDirectory);
+            CopyIntermediaryAssemblies(Application.dataPath.Replace("Assets", "NuGet/Plugins"));
+            UpdateMetaFiles();
+        }
+
+
+
+        private static void CopyIntermediaryAssemblies(string outputPath)
+        {
+            List<DirectoryInfo> outputDirectories = new List<DirectoryInfo>();
+            string playerDataCachePath = Application.dataPath.Replace("Assets", "Library/PlayerDataCache");
+            DirectoryInfo playerDataCahce = new DirectoryInfo(playerDataCachePath);
+            DirectoryInfo[] dllStores = playerDataCahce.GetDirectories("*", SearchOption.TopDirectoryOnly);
+            string subfolderName = string.Empty;
+            foreach(KeyValuePair<string, string> sourceToOutputPair in sourceToOutputFolders)
+            {
+                DirectoryInfo directory = new DirectoryInfo(Application.dataPath.Replace("Assets", sourceToOutputPair.Key));
+                if (!directory.Exists)
+                {
+                    throw new InvalidDataException($"The source directory {sourceToOutputPair.Key} does not exist.");
```

Comment:
>The source directory [](start = 53, length = 20)

Phrase it as: "The required platform intermediary build directory {sourceToOutputPair.Key} does not exist. Was the build succesful?"